tests/test_quasiharmonic_lammps.py

```diff
@@ -59,16 +59,16 @@ def test_calc_phonons(self):
         self.assertTrue(tp_collect_dict["free_energy"][-1] > -2.7)
         self.assertTrue(tp_collect_dict["entropy"][0] < 0.1)
         self.assertTrue(tp_collect_dict["entropy"][0] > 0.0)
-        self.assertTrue(tp_collect_dict["entropy"][-1] < 273)
-        self.assertTrue(tp_collect_dict["entropy"][-1] > 272)
+        self.assertTrue(tp_collect_dict["entropy"][-1] < 270)
+        self.assertTrue(tp_collect_dict["entropy"][-1] > 269)
```

Comment:
So, I still remain somewhat uncomfortable testing against hard floats for the physical properties, but I don't expect that to be treated in this PR.

However, I would suggest to start replacing the dual >/< tests with [`self.assertAlmostEqual`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertAlmostEqual), which will improve test readability. IMO this can be done piecemeal as you go through the tests rather than as a monolithic change, but that's up to you.

```suggestion
        self.assertAlmostEqual(tp_collect_dict["entropy"][-1], 269.5, delta=0.5)
```

Reply:
I agree with the suggestion, still that is something I would prefer to move to a separate pull request and do it for a series of tests at once.